### ISSUE TITLE
The function you are calling is not pressent in your version of GTK+.

### ISSUE THREAD

**Issue by Prajithp** [gnunn1/tilix#1106]
I have compiled tilix on my linux mint 17.3, but getting this error when executing tilix binary. 

```
firefly tilix # LD_LIBRARY_PATH=/opt/glib/lib/:/opt/gtk314/lib/:/opt/pango/lib/:/opt/atk/lib/ ./tilix 
2017-09-03T01:19:46.875:app.d:main:37 CWD = /root/tilix
2017-09-03T01:19:46.875:app.d:main:43 No PWD environment variable found
2017-09-03T01:19:46.875:app.d:main:52 UHD = /root
2017-09-03T01:19:46.875:app.d:main:56 args[0]=./tilix

** (tilix:27947): WARNING **: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files
2017-09-03T01:19:46.884:app.d:main:102 Starting tilix with 1 arguments...
2017-09-03T01:19:46.884:app.d:main:104 arg[0] = ./tilix
2017-09-03T01:19:46.884:app.d:main:126 No tilix UUID found
2017-09-03T01:19:46.884:app.d:main:141 Creating app
object.Error@(0): The function you are calling is not pressent in your version of GTK+.
----------------
??:? extern (C) void gtkd.Loader.Linker.unsupportedSymbol() [0xe737a0]
??:? void gio.Application.Application.addMainOption(immutable(char)[], char, gtkc.glibtypes.GOptionFlags, gtkc.glibtypes.GOptionArg, immutable(char)[], immutable(char)[]) [0xcfac69]
??:? void gx.tilix.application.Tilix.addOptions() [0xc41ff2]
??:? gx.tilix.application.Tilix gx.tilix.application.Tilix.__ctor(bool) [0xc428b8]
??:? _Dmain [0xba7722]
``` 
```
firefly tilix # PKG_CONFIG_PATH=/opt/gtk314/lib/pkgconfig/ pkg-config --modversion gtk+-3.0 
3.14.0
```

**Comment by gnunn1** [comment]
I think you may have something mis-configured on your system, I just tried master on GTK 3.14 (CentOS 7.3) and it worked fine for me. Here's the versions reported when I run ```tilix --version```, what versions do you see on Mint?

```
[gnunn@localhost sf_tilix]$ ./tilix --version
Versions
	Tilix version: 1.6.5-0.0.2
	VTE version: 0.42
	GTK Version: 3.14.13

Tilix Special Features
	Notifications enabled=0
	Triggers enabled=0
	Badges enabled=0
```

**Comment by Prajithp** [comment]
Thank you. I have managed to fix this issue by installing gtk-3.16. But, still not sure why it doesn't work with gtk-3.14. 

Also, is there a way to avoid calling LD_LIBRARY_PATH every time and compile tilix with alternative gtk?

firefly ~ # LD_LIBRARY_PATH=/opt/gtk314/lib/  tilix --version
2017-09-04T10:12:32.207:app.d:main:37 CWD = /root
2017-09-04T10:12:32.210:app.d:main:43 No PWD environment variable found
2017-09-04T10:12:32.210:app.d:main:52 UHD = /root
2017-09-04T10:12:32.210:app.d:main:56 args[0]=tilix
2017-09-04T10:12:32.210:app.d:main:56 args[1]=--version

** (tilix:4826): WARNING **: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files
2017-09-04T10:12:32.282:app.d:main:102 Starting tilix with 2 arguments...
2017-09-04T10:12:32.282:app.d:main:104 arg[0] = tilix
2017-09-04T10:12:32.282:app.d:main:104 arg[1] = --version
Versions
        Tilix version: 1.6.5-0.0.2
        VTE version: 0.48
        GTK Version: 3.16.0

Tilix Special Features
2017-09-04T10:12:32.328:vte.d:checkVTEFeature:109 Background draw disabled
        Notifications enabled=0
        Triggers enabled=0
        Badges enabled=0

**Comment by gnunn1** [comment]
I'm not aware of a way to use alternative GTKs other then LD_LIBRARY_PATH. If I had to guess, I'd say there is something wrong with your 3.14 configuration since it seems to be failing Application.addMainOption and that's been around a long time.
